controller.go

```diff
@@ -372,6 +372,12 @@ func syncPV(pv *PV) {
 				}
 			} else {
 				// The PV must have been created with this ptr; leave it alone.
```

Comment:
Oops, we raced

On Wed, Apr 13, 2016 at 12:58 PM, Paul Morie <notifications@github.com>
wrote:

> In controller.go
> <https://github.com/pmorie/pv-haxxz/pull/4#discussion_r59614622>:
>
> > @@ -372,6 +372,12 @@ func syncPV(pv *PV) {
> >  				}
> >  			} else {
> >  				// The PV must have been created with this ptr; leave it alone.
>
> Remove this line now? @jsafrane <https://github.com/jsafrane>
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/pmorie/pv-haxxz/pull/4/files/f8c2d4cc22beae7a53d3d5ffb6ee82825562c423#r59614622>
>